src/charm.py

```diff
@@ -42,6 +63,14 @@ def __init__(self, *args):
         self.framework.observe(self.database.on.database_created, self._on_database_changed)
         self.framework.observe(self.database.on.endpoints_changed, self._on_database_changed)
 
+        # Admin ingress events
```

Comment:
I don't think those comments add to the understanding

Reply:
For me they are useful when I am trying to skim through the registered hooks of an operator that has many of them. If you wish I can remove these comments.

Reply:
I don't insist on it, honestly it doesn't make much of a difference, but if we want to have comments to separate the event types, we should probably add some to the database events as well to at least be consistent.